### ISSUE TITLE
Error while loading primes @3.0.14 on Windows

### ISSUE THREAD

**Issue by kyuhyongpark** [hklarner/pyboolnet#117]
Hello, I updated to pyboonet to version `3.0.14` and encountered an error that didn't occur with `3.0.11`.

I have set up a new virtual environment with Python 3.11.4 on Windows and installed `pyboolnet@3.0.14`
```
pip3 install git+https://github.com/hklarner/pyboolnet@3.0.14
```

I reproduced the error with the below script:
```
from pyboolnet.external.bnet2primes import bnet_text2primes

primes = bnet_text2primes("""
targets, factors
a, b
b, c
c, a""")

print(primes)
```

```
Traceback (most recent call last):
  File ".\bug_test.py", line 3, in <module>
    primes = bnet_text2primes("""
             ^^^^^^^^^^^^^^^^^^^^
  File ".\.venv\Lib\site-packages\pyboolnet\external\bnet2primes.py", line 27, 
in bnet_text2primes
    proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".\Python\Python311\Lib\subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File ".\Python\Python311\Lib\subprocess.py", line 1538, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

Using the previous version `3.0.11`,

```
pip3 install --force-reinstall git+https://github.com/hklarner/pyboolnet@3.0.11
```

returns

```
{'a': [[{'b': 0}], [{'b': 1}]], 'b': [[{'c': 0}], [{'c': 1}]], 'c': [[{'a': 0}], [{'a': 1}]]}
```

In case it helps, with `3.0.12`, I get

```
could not read NuSMV keywords: exception=[Errno 2] No such file or directory: '.\\.venv\\Lib\\site-packages\\pyboolnet\\binaries\\nusmvkeywords.json'
Traceback (most recent call last):
  File ".\.venv\Lib\site-packages\pyboolnet\helpers.py", line 52, in read_nusmv_keywords_or_exit
    return read_nusmv_keywords()
           ^^^^^^^^^^^^^^^^^^^^^
  File ".\.venv\Lib\site-packages\pyboolnet\helpers.py", line 46, in read_nusmv_keywords
    with open(fname) as fp:
         ^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '.\\.venv\\Lib\\site-packages\\pyboolnet\\binaries\\nusmvkeywords.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".\bug_test.py", line 1, in <module>
    from pyboolnet.external.bnet2primes import bnet_text2primes
  File ".\.venv\Lib\site-packages\pyboolnet\__init__.py", line 11, in <module> 
    NUSMV_KEYWORDS = read_nusmv_keywords_or_exit()
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".\.venv\Lib\site-packages\pyboolnet\helpers.py", line 55, in read_nusmv_keywords_or_exit
    raise Exception
Exception
```

With `3.0.13` I get the same error as `3.0.14`

**Comment by hklarner** [comment]
It seems that your pyboolnet installation does include binary for the program `bnet2prime`. It probably affects all windows users. You can determine the path it is looking for with this call:
```python
from pyboolnet import find_command

print(find_command("bnet2prime"))
```

Can you confirm that this path does not exist for you?

With pyboolnet comes a command line tool that is also called `pyboolnet`. In Linux I can call it in the terminal:
```
$ pyboolnet
pyboolnet version: 3.0.14
```

Can you confirm that this works for you as well, in a windows terminal?

If so, can you post the output of this command:
```
$ pyboolnet check-dependencies
```

A temporary workaround for you is to manually copy the binaries from github into that location. You can download the binaries here
- https://github.com/hklarner/pyboolnet/blob/master/binaries/win64/BNetToPrime/BNetToPrime_win64.exe

I'd appreciate it if you h

**Comment by kyuhyongpark** [comment]
> It seems that your pyboolnet installation does include binary for the program `bnet2prime`. It probably affects all windows users. You can determine the path it is looking for with this call:
> 
> ```python
> from pyboolnet import find_command
> 
> print(find_command("bnet2prime"))
> ```
> 
> Can you confirm that this path does not exist for you?

This returns 
```
..\.venv\Lib\site-packages\pyboolnet\binaries\BNetToPrime\BNetToPrime
```
But this directory does not exist.

> 
> With pyboolnet comes a command line tool that is also called `pyboolnet`. In Linux I can call it in the terminal:
> 
> ```
> $ pyboolnet
> pyboolnet version: 3.0.14
> ```
> 
> Can you confirm that this works for you as well, in a windows terminal?
> 

It does work.

> If so, can you post the output of this command:
> 
> ```
> $ pyboolnet check-dependencies
> ```

This returns an error. I wonder why the gringo command says 'linux'.

```
pyboolnet version: 3.0.14
'CONDA' in os.environ=False
platform.system()='Windows'
INFO gringo command = ..\.venv\Lib\site-packages\pyboolnet\binaries\gringo-4.4.0-x86-linux\gringo --help
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "..\.venv\Scripts\pyboolnet.exe\__main__.py", line 7, in <module>       
  File "..\.venv\Lib\site-packages\click\core.py", line 1157, in __call__      
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "..\.venv\Lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "..\.venv\Lib\site-packages\click\core.py", line 1719, in invoke
    rv.append(sub_ctx.command.invoke(sub_ctx))
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "..\.venv\Lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "..\.venv\Lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "..\.venv\Lib\site-packages\pyboolnet\cli\commands\check_dependencies.py", line 11, in check_dependencies
    check_all_dependencies()
  File "..\.venv\Lib\site-packages\pyboolnet\cli\check_dependencies.py", line 19, in check_all_dependencies
    check_gringo_responds()
  File "..\.venv\Lib\site-packages\pyboolnet\cli\check_dependencies.py", line 34, in check_gringo_responds
    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "..\Python\Python311\Lib\subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "..\Python\Python311\Lib\subprocess.py", line 1538, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

> 
> A temporary workaround for you is to manually copy the binaries from github into that location. You can download the binaries here
> 
> * https://github.com/hklarner/pyboolnet/blob/master/binaries/win64/BNetToPrime/BNetToPrime_win64.exe

I am trying this out right now. Thank you.

**Comment by hklarner** [comment]
To reproduce your problem, I've created a fresh clone of the repo and called `pip install .` in the root directory. I am on Linux, so this copies the correct binaries to the correct location, namely `./binaries/linux/` to `./pyboolnet/binaries/`. Pyboolnet uses 
```python
# setup.py

THIS_OS = platform.system()
```
to decide which binaries to copy. If I change the source code to `THIS_OS = "Windows"` and pip install again, then the correct windows binaries are copied.

So I can not reproduce your error so far (but also, I am not on a Windows machine, so I don't really know what happens on Windows).

Is there anything particular to your OS or pyboolnet installation? Are you really just using a plain Windows machine?

> This returns an error. I wonder why the gringo command says 'linux'.

This is indeed very suspicious. Could it be that you previously had a Linux installation on your machine?

**Comment by kyuhyongpark** [comment]
> Can you install that branch and confirm that the Windows binaries are shipped?
> 
> ```
> $ pip3 install git+https://github.com/hklarner/pyboolnet@bugfix/117-windows-binaries
> ```

I confirmed that the Windows binaries are shipped properly. All the previous test scripts run fine as well.
I'll make a pool request. Thank you for the prompt fix!

**Comment by hklarner** [comment]
Sorry, I meant that you create your own branch and your own commit (but with the same changes as in my bugfix branch).

I have just merged your PR and then realized that I am still the author of the commit.

So I have reverted the develop branch and you can do it again with your own commit.

**Comment by kyuhyongpark** [comment]
I'm sorry I misunderstood as I am not familiar with the common practice. I made the commit and the pull request again. I verified again that the binaries are shipped properly. Thank you for your patience.

**Comment by hklarner** [comment]
Great, thanks a lot for your contribution! :tada: It is released with tag 3.0.15:
- https://github.com/hklarner/pyboolnet/discussions/90?sort=new#discussioncomment-11128020
